modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/searcher/SearchResponse.java

```diff
@@ -30,11 +30,15 @@
 import java.util.stream.Stream;
 
 /**
- * Holds various outcomes returned when a search is performed.
+ * Contains the full response of an executed search, as returned from the search
+ * engine.
  *
- * @author André de Oliveira
+ * <p>
+ * i The exact response format and the contents of the response depends on the
```

Comment:
what does the "i" do here?

Reply:
Oh, that - err, it was to see if you really reviewed this closely enough. Well done: :medal_sports: 

:smirk:

Reply:
haha. well my speed reading skills are improving...